domains/auth/Auth.API/API/Controllers/AuthController.cs

```diff
@@ -30,29 +30,29 @@ public AuthController(ILogger<AuthController> logger, IOidcProviders oidcProvide
         [Required] string feUrl,
         [Required] string returnUrl)
     {
-        AuthState state = new AuthState()
+        var state = new AuthState()
```

Comment:
Parentheses can be omitted when initializing in this way. Can this be automated?

Reply:
There appears to be only [two rules](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0047-ide0048) for parentheses and them both seem irrelevant in regards to this. If we should find more rules, let us add them in another PR.